src/components/AnonymousChat/child/ChannelImage.tsx

```diff
@@ -56,13 +57,13 @@ const ChannelImage = ({
       display: 'flex',
       justifyContent: 'center',
       alignItems: 'center',
-      backgroundColor: COLORS.anon_primary
+      backgroundColor: COLORS.anon_secondary
     },
     myPostNotificationImageContainer: {
-      backgroundColor: isAnonymousTab ? COLORS.anon_primary : COLORS.signed_secondary
+      backgroundColor: isAnonymousTab ? COLORS.anon_secondary : COLORS.signed_secondary
     },
     anonPmNotificationImageContainer: {
-      backgroundColor: COLORS.anon_primary
+      backgroundColor: COLORS.anon_secondary
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

Similar to the previous comment, the `backgroundColor` property has been added to several style objects and is hardcoded to specific colors. If these colors are meant to be dynamic, consider passing them as props or retrieving them from a context/provider.
```diff
-      backgroundColor: COLORS.anon_secondary
+      backgroundColor: props.backgroundColor || COLORS.anon_secondary
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->